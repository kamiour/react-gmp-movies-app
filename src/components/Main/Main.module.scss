@import 'src/scss/variables';

.main {
  margin-top: 10px;
  padding: 0 #{$layout-side-padding};
  background-color: $color_nero;
  padding-bottom: 73px;
}
