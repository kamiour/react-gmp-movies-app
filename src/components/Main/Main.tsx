import { PropsWithChildren } from 'react';
// import './Main.scss';

function Main({ children }: PropsWithChildren<{}>) {
  return <div className="main">{children}</div>;
}

export default Main;
