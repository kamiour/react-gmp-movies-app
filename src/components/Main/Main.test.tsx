import { render } from '@testing-library/react';
import Main from './Main';

describe('Main', () => {
  it('should match Main snapshot', () => {
    const { asFragment } = render(<Main>Main</Main>);
    expect(asFragment()).toMatchSnapshot();
  });
});
