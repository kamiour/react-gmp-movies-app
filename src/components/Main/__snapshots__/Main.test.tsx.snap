// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should match Main snapshot 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    Main
  </div>
</DocumentFragment>
`;
